From 17e8f1fc67b05740b97ab0d6b3adc7f607892709 Mon Sep 17 00:00:00 2001
From: Cyril Hrubis <chrubis@suse.cz>
Date: Tue, 14 May 2013 15:14:03 +0200
Subject: [PATCH] openposix/.../sigaddset/1-3: Fix uninitialized variable.

My last fix in this testcase introduced uninitialized variable, sorry.

Signed-off-by: Cyril Hrubis <chrubis@suse.cz>
---
 .../conformance/interfaces/sigaddset/1-3.c         |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/testcases/open_posix_testsuite/conformance/interfaces/sigaddset/1-3.c b/testcases/open_posix_testsuite/conformance/interfaces/sigaddset/1-3.c
index 144d2ac..bb80da7 100644
--- a/testcases/open_posix_testsuite/conformance/interfaces/sigaddset/1-3.c
+++ b/testcases/open_posix_testsuite/conformance/interfaces/sigaddset/1-3.c
@@ -31,7 +31,8 @@ static const int sigs[] = {
 int main(void)
 {
 	sigset_t signalset;
-	int i, err;
+	unsigned int i;
+	int err = 0;
 
 	if (sigemptyset(&signalset) == -1) {
 		perror("sigemptyset failed -- test aborting\n");
-- 
1.7.8.6

